AirCasting/Map/LocationTracker.swift

```diff
@@ -45,6 +45,7 @@ final class CoreLocationTracker: NSObject, LocationTracker, LocationAuthorizatio
             locationManager.startUpdatingLocation()
         }
         locationStartReference += 1
+        Log.info("Started location tracking (refcount: \(locationStartReference))")
```

Comment:
It shouldn't flood us, and I think it might be helpful to see. In particular the `refcount` here can be a good debugging tool (we can track which actions triggered location tracking and if each of them has a corresponding stopTracking call)